libvirt/tests/src/migration/migration_with_disk/migration_with_vhostuser.py

```
@@ -0,0 +1,209 @@
+    """
+    # Create backend image in source host
+    libvirt.create_local_disk("file", image_path, size="100M")
+    chown_cmd = "chown qemu:qemu %s" % image_path
+    process.run(chown_cmd, ignore_status=False, shell=True)
+    # Start vhost sock service in source host
+    cmd_output = process.run(start_sock_service_cmd, ignore_status=False, shell=True).stdout_text.strip()
+    # Set SELinux context in source host
+    ch_seccontext_cmd = "chcon -t svirt_image_t %s" % sock_path
+    process.run(ch_seccontext_cmd, ignore_status=False, shell=True)
+    set_bool_mmap_cmd = "setsebool domain_can_mmap_files 1 -P"
+    process.run(set_bool_mmap_cmd, ignore_status=False, shell=True)
```

Ruff (0.14.6)

Function call with `shell=True` parameter identified, security issue

(S604)